tests/test_notanorm.py

```diff
@@ -1035,18 +1048,21 @@ def test_joinq_model_changes(db_sqlite_notmem):
 
     assert db.select(db.join("a", "b", bid="bid"))
 
-    file = db.uri.split(":", 1)[1]
-
-    dbx = sqlite3.connect(file)
-
     # other process has changed things!
-    dbx.execute("drop table b")
-    dbx.execute("create table b (b_id integer primary_key, c_id integer)")
-    dbx.execute("insert into b values(1, 1)")
+    db._conn().execute("drop table b")
+    db._conn().execute("create table b (b_id integer primary_key, c_id integer)")
+    db._conn().execute("insert into b values (1, 1)")
 
     with pytest.raises(err.OperationalError):
         db.select(db.join("a", "b", bid="b_id"))
 
+    # explicit table names is always ok
+    assert db.select(db.join("a", "b", a__bid="b__b_id"), ["a.aid", "b.b_id"])
+
+    # asterisk is cool too
+    assert db.select(db.join("a", "b", a__bid="b__b_id"), ["a.*"])[0].aid == 1
```

Comment:
your use case colin

Reply:
it works because even though it get the cols it doesnt actually need/use them 

there's a test that proves it works in this PR